### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by spydon**
# Description

Removing the emoji title and organize things a little bit.

## Type of change

Please delete options that are not relevant.

- [x] Doc update

# Checklist:

If something is unclear, please submit the PR anyways and ask about what you thought was unclear.

- [x] This branch is based on `develop`
- [x] This PR is targeted to merge into `develop` (not `master`)
- [x] I have made corresponding changes to the documentation
- [x] The continuous integration (CI) is passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
+We also have a [FAQ](FAQ.md), so please search for your question there first.
 
 ## Goals
 
 The goal of this project is to provide a complete set of out-of-the-way solutions for the common problems every game developed in Flutter will share.
 
-Currently it provides you with: a few utilities, images/sprites/sprite sheets, audio, a physics engine, a game loop and a component/object system.
+Currently it provides you with: 
+ - a game loop 
+ - a physics engine
```
Comment (erickzanardo) [inline README.md:21]: Suggestion: bundles a physics engine (box2d)

File: README.md [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
+ - audio support
+ - effects and particles
+ - gesture and input support
+ - images, sprites and sprite sheets
+ - a component/object system
```
Comment (erickzanardo) [inline README.md:26]: Basic riveapp support

File: README.md (hunk 1 above)
Comment (spydon) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (spydon) [reply README.md:26]: Fixed

File: README.md [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
+Currently it provides you with: 
+ - a game loop 
+ - bundles a physics engine (box2d)
+ - audio support
+ - effects and particles
+ - gesture and input support
+ - images, sprites and sprite sheets
+ - a component/object system
+ - basic Rive support
+ - and a few utilities to make development easier
```
Comment (luanpotter) [inline README.md:28]: how about:

`a few [other] utilities`

File: README.md [hunk 4]
```diff
@@ -1,38 +1,47 @@
...
 
-Currently it provides you with: a few utilities, images/sprites/sprite sheets, audio, a physics engine, a game loop and a component/object system.
+Currently it provides you with: 
+ - a game loop 
+ - bundles a physics engine (box2d)
+ - audio support
+ - effects and particles
+ - gesture and input support
+ - images, sprites and sprite sheets
+ - a component/object system
```
Comment (luanpotter) [inline README.md:26]: how about moving this below game loop? they are related and it's a very important feature

File: README.md (hunk 3 above)
Comment (spydon) [reply README.md:28]: Or maybe just remove this line, it doesn't really add much?

File: README.md (hunk 4 above)
Comment (spydon) [reply README.md]: Fixed!

**Pull request merged**